config/printer-biqu-b1-se-plus.cfg

```diff
@@ -0,0 +1,219 @@
+# is used, if this is the case compile for this with a "32KiB bootloader"
+# You will need to check the chip on your board to identify which you have.
+#
+# The "make flash" command does not work on the SKR 2. Instead,
+# after running "make", copy the generated "out/klipper.bin" file to a
+# file named "firmware.bin" on an SD card and then restart the SKR 2
+# with that SD card.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+# Note: The initial revision of this board has a flaw that can cause
+# damage to itself and other boards. Be sure to verify the board is
+# not impacted by this flaw before using it.
+
+[mcu]
+serial: /dev/serial/by-id/usb-Klipper_stm32f407xx_1D0039000F47393438343535-if00
+
+########################################
+# Stepper X Pins and TMC2208 configuration
+########################################
+[stepper_x]
+step_pin: PE2
+dir_pin: !PE1
+enable_pin: !PE3
+microsteps: 16
+rotation_distance: 40.12
+endstop_pin: !PC1
+position_endstop: 0
+position_max: 310
+homing_speed: 50
+#homing_positive_dir: false
+#homing_retract_dist: 5
+
+[tmc2208 stepper_x]
+uart_pin: PE0
+run_current: 0.800
+hold_current: 0.400
+stealthchop_threshold: 999999
+
+########################################
+# Stepper Y Pins and TMC2208 configuration
+########################################
+[stepper_y]
+step_pin: PD5
+dir_pin: PD4
+enable_pin: !PD6
+microsteps: 16
+rotation_distance: 40.28
+endstop_pin: !PC3
+position_endstop: 0
+position_max: 310
+homing_speed: 50
+#homing_retract_dist: 5
+
+[tmc2208 stepper_y]
+uart_pin: PD3
+run_current: 0.800
+hold_current: 0.400
+stealthchop_threshold: 999999
+
+########################################
+# Stepper Z Pins and TMC2208 configuration
+########################################
+[stepper_z]
+step_pin: PA15
+dir_pin: PA8
+enable_pin: !PD1
+microsteps: 16
+rotation_distance: 8.13656
+endstop_pin: probe:z_virtual_endstop
+homing_speed: 10
+second_homing_speed: 1
+position_min: -2
+position_max: 340
+
+[tmc2208 stepper_z]
+uart_pin: PD0
+run_current: 0.800
+hold_current: 0.400
+stealthchop_threshold: 999999
+
+########################################
+# Extruder Pins and TMC2208 configuration
+########################################
+[extruder]
+step_pin: PD15
+dir_pin: !PD14
+enable_pin: !PC7
+microsteps: 16
+rotation_distance: 34.2152 # Calibrar - ver https://www.klipper3d.org/Rotation_Distance.html
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PB3
+sensor_type: Generic 3950
+sensor_pin: PA2 #thermistor pin
+control: pid
+pid_Kp: 22.2
+pid_Ki: 1.08
+pid_Kd: 114
+min_temp: 0
+max_temp: 250
+
+[tmc2208 extruder]
+uart_pin: PC6
+run_current: 0.800
+hold_current: 0.300
+stealthchop_threshold: 999999
+########################################
+
+########################################
+# Heater Bed Pins
+########################################
+[heater_bed]
+heater_pin: PD7
+sensor_type: Generic 3950
+sensor_pin: PA1
+control: pid
```

Comment:
Might be best to remove the PID section, and let the user decide later if they want to use it for their bed.

Reply:
I can delete it, but then when klipper checks the configuration it will give an error because these parameters are missing.  Is it really convenient to delete?

Reply:
I am wrong on that one, sorry.